internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/test/table.go

```diff
@@ -110,7 +110,7 @@ func MockTableStorageAPICalls(tb testing.TB, transport *httpmock.MockTransport)
 	// Create table job - ok
 	transport.RegisterResponder(
 		http.MethodGet,
-		`=~/v2/storage/jobs/.+$`,
+		`=~/v2/storage/jobs/1.*`,
```

Comment:
Wait why the `1`? What if the job begins with `2`?

Reply:
I don't understand what you're talking about. I'll need some clarification on this... Though it seems to be only for a test so I guess it's not all that important.

Reply:
Okay I think I understand this slightly better now. But where is the id for a table job decided? How do you know it will always begin with a `1`? I was unable to find that.